Gemfile.lock

```diff
@@ -82,7 +83,10 @@ PATH
       addressable
       aws-sdk-s3 (~> 1.91)
       aws-sdk-secretsmanager (~> 1.46)
+      chef-config (= 19.0.31)
```

Comment:
This is not supposed to be downgraded manually. @ahasunos  Looks like some mistake during resolving conflicts?

Reply:
yup! Mistake during resolving conflicts. Let me fix it soon.